lib/ansible/modules/cloud/digital_ocean/_digital_ocean_sshkey_facts.py

```diff
@@ -91,6 +95,8 @@ def main():
         supports_check_mode=False,
     )
 
+    module.deprecate("The 'digital_ocean_sshkey_facts' module has been deprecated, use the new 'digital_ocean_sshkey_info'", version='2.13')
```

Comment:
```suggestion
    module.deprecate("The 'digital_ocean_sshkey_facts' module has been deprecated, use the new 'digital_ocean_sshkey_info' module", version='2.13')
```
To make this a complete sentence :)

Reply:
😓